helpers/deploymentConfig.ts

```diff
@@ -309,6 +317,13 @@ export const chainlinkFeed: Config = {
     USDT: "0xF83696ca1b8a266163bE252bE2B94702D4929392",
     OP: "0x8907a105E562C9F3d7F2ed46539Ae36D87a15590",
   },
+  opmainnet: {
+    WBTC: "0xD702DD976Fb76Fffc2D3963D037dfDae5b04E593",
+    WETH: "0x13e3Ee699D1909E989722E753853AE30b17e08c5",
+    USDC: "0x16a9FA2FDa030272Ce99B29CF780dFA30361E0f3",
+    USDT: "0xECef79E109e997bCA29c1c0897ec9d7b03647F5E",
+    OP: "0x0D276FC14719f9292D5C1eA2198673d1f4269246",
+  },
 };
 
 export const redstoneFeed: Config = {
```

Comment:
Shouldn't we add XVS redstone feed here?

Reply:
https://github.com/VenusProtocol/oracle/pull/225/commits/2e8b8692ea8a48d0f10c0ad70a92ae4a38be3726